charts/api/templates/_helpers.tpl

```diff
@@ -56,9 +56,9 @@ Common lists of environment variables
   value: {{ .Values.wbstack.wikiDbProvisionVersion | quote }}
 - name: WBSTACK_WIKI_DB_USE_VERSION
   value: {{ .Values.wbstack.wikiDbUseVersion | quote }}
-{{- if .Values.wbstack.maxWikisPerUser }}
```

Comment:
Not sure I follow this part of the change. Is this inadvertently removing this env var?